src/main/java/frc/robot/Commands/ArmCommand.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.Commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Subsystems.ArmSubsystem;
+
+public class ArmCommand extends CommandBase {
```

Comment:
Are we using this?

Reply:
not currently, but it's an easy alternative to calling moveArm() inside a run command, and will be better to build upon as more code is added to the arm subsystem